CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 1.4.20 (Nov 2020)
+  * XCode 12 support
+  * Completely reworked escape analysis for object allocation
+  * Use ForeignException wrapper to handle native exceptions ([KT-35056](https://youtrack.jetbrains.com/issue/KT-35056))
```

Comment:
This issue is internal (used only as a reminder). Please use this one instead: https://github.com/JetBrains/kotlin-native/issues/3553